ui/src/app/components/object-detail/object-detail.component.ts

```diff
@@ -529,8 +621,33 @@ export class ObjectDetailComponent implements OnInit {
         data: { message: 'Add columns to the primary key for saving', type: 'error' },
         maxWidth: '500px',
       })
+    } else {
+      let interleaveTable = this.tableInterleaveWith(this.currentObject?.name!)
+      if (interleaveTable != '' && this.isPKFirstOrderModified(this.currentObject?.name!)) {
+        const dialogRef = this.dialog.open(ConfirmationDialogComponent, {
+          data: {
+            message:
+              'Proceeding the update will removed interleaving between ' +
```

Comment:
Proceeding with the update will remove interleaving between

Reply:
Updated